lib/srv/desktop/rdp/rdpclient/src/rdpdr.rs

```diff
@@ -143,6 +146,28 @@ impl TeleportRdpdrBackend {
         Ok(())
     }
 
+    fn handle_list_readers(
+        &mut self,
+        req: DeviceControlRequest<ScardIoCtlCode>,
+        _call: ListReadersCall,
+    ) -> PduResult<()> {
+        self.write_rdpdr_dev_ctl_resp(
+            req,
+            Box::new(ListReadersReturn::new(
```

Comment:
I wonder if it makes sense to extract this whole `Box::new` (and other static messages) to `#[dynamic]` so we can reuse them across calls

Reply:
It would work as an optimization in the future, I don't think it's worth concerning ourselves with now.